<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinecraftLauncher</class>
 <widget class="QMainWindow" name="MinecraftLauncher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GMCL α-1</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/resources/reinforced_deepslate.png</normalon>
   </iconset>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QToolButton" name="home_btn">
        <property name="text">
         <string>主页</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/resources/home.svg</normalon>
         </iconset>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="core_btn">
        <property name="text">
         <string>核心</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="user_btn">
        <property name="text">
         <string>账户</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/resources/user.svg</normalon>
         </iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_4">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QToolButton" name="quit_btn">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/resources/exit.svg</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="setting_btn">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/resources/settings.svg</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="home">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>选择版本</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::AutoText</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox"/>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton_7">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normalon>:/resources/sliders.svg</normalon>
                 </iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="run_btn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>启动游戏</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="core"/>
      <widget class="QWidget" name="user"/>
      <widget class="QWidget" name="page_4"/>
      <widget class="QWidget" name="setting">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="game_path">
          <item>
           <widget class="QLabel" name="game_path_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>游戏根目录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="game_path_lineEdit">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>请输入游戏根目录（.minecraft）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="game_select">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="gmcl.qrc">
              <normaloff>:/resources/folder-plus.svg</normaloff>:/resources/folder-plus.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="java_path">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="java_path_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>java路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="java_path_lineEdit">
            <property name="placeholderText">
             <string>请输入Java路径（java.exe）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="java_select">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="gmcl.qrc">
              <normaloff>:/resources/folder-plus.svg</normaloff>:/resources/folder-plus.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="username">
          <item>
           <widget class="QLabel" name="username_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>用户名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="username_lineEdit">
            <property name="placeholderText">
             <string>请输入用户名（暂时仅支持离线登录）</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="men">
          <item>
           <widget class="QLabel" name="maxmen_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>最大内存</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="gmcl.qrc"/>
 </resources>
 <connections/>
</ui>
